internal/driver/fetch_test.go

```diff
@@ -423,9 +423,15 @@ func TestHttpsInsecure(t *testing.T) {
 	if len(p.SampleType) == 0 {
 		t.Fatalf("fetchProfiles(%s) got empty profile: len(p.SampleType)==0", address)
 	}
-	if runtime.GOOS == "plan9" {
+	switch runtime.GOOS {
+	case "plan9":
 		// CPU profiling is not supported on Plan9; see golang.org/issues/22564.
 		return
+	case "darwin":
+		if runtime.GOARCH == "arm" || runtime.GOARCH == "arm64" {
+			// CPU profiling is not supported on OS; see golang.org/issues/22612.
```

Comment:
Is there a more useful bug to link, the one which actually tracks the profiling issue on iOS rather than points to the pprof's test breakage? Like we list golang.org/issues/22564 for Plan9.

Reply:
As per my comment above, I don't there is an issue with the basic CPU profiling machinery on iOS.